src/component/home.js

```diff
@@ -22,7 +26,26 @@ export default function Home() {
               <li className="card-vertical" key={video._id}>
                 <div className="card__primary-action card__primary-action-column card__primary-action-vertical">
                   <span className="material-icons-outlined badge-up-right-corner card-badge card-badge-vertical">
-                    <AiTwotoneLike />
+                    {likeState.likeVideos &&
+                    likeState.likeVideos.some(
+                      (item) => video._id === item._id
+                    ) ? (
```

Comment:
create a variable 

```js 
const isLiked = likeState.likeVideos.some(
                      (item) => video._id === item._id
                    ) 
```

Reply:
Thanks for suggestion . I will do it in next PR.